OpenDBDiff/Front/PrincipalForm.Designer.cs

```diff
@@ -658,9 +623,24 @@ private void InitializeComponent()
             this.PanelActions.Name = "PanelActions";
             this.PanelActions.Size = new System.Drawing.Size(98, 483);
             this.PanelActions.TabIndex = 17;
-            //
+            // 

```

Comment:
Why did you move this block?

Reply:
Probably just a Visual Studio .designer.cs randomness due to changes in control order... I vote for letting people stomp all over order in the .designer.cs files as long as nothing breaks.

Reply:
In that case, when 2 people are working on different branches you're bound to get conflicts, or at least a corrupt designer file.

Also, as I've just experienced, extremely difficult to identify the actual logical change and revert it.

Reply:
Fair enough. It sounds like the solution is "WPF"! Manually cleaning up auto-generated order changes is decade+ struggle that discourages fixes to the current UI, but I don't see anything that can help with it.

I appreciate you taking the time to look things over. It sounds like this will be tedious for at least one person (contributor or reviewer) and in this case seems to be both. Keep up the good work!

https://www.google.com/search?q=.net+winforms+designer+source+control+conflict

http://blog.plasticscm.com/2013/11/the-sortresx-tool.html